data/tlg0551/tlg009/__cts__.xml

```diff
@@ -2,11 +2,12 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0551.tlg009" xml:lang="grc" groupUrn="urn:cts:greekLit:tlg0551">
     <ti:title xml:lang="eng">Punic Wars</ti:title>
     <ti:edition xml:lang="grc" workUrn="urn:cts:greekLit:tlg0551.tlg009" urn="urn:cts:greekLit:tlg0551.tlg009.perseus-grc2">
-        <ti:label xml:lang="lat">Punic Wars, Appiani Historia romana Volumen Prius</ti:label>
-        <ti:description xml:lang="eng">Appianus, of Alexandria, creator; Mendelssohn, Ludwig, 1852-1896, editor</ti:description>
+        <ti:label xml:lang="grc">ΛΙΒΥΚΗ</ti:label>
```

Comment:
Either we should stick with the English or I can go back and find the lowercase for these. It will take a few minutes.